build_for_ultimaker.sh

```diff
@@ -39,7 +39,7 @@ setup_emulation_support()
         fi
```

Comment:
By the way, comment on your previous pull request:

grep interpreter "/proc/sys/fs/binfmt_misc/qemu-arm" | sed 's/interpreter //'

Reply:
excellent, that's almost the one liner i was looking for and sed is part of busybox.
`
busybox sed 's/interpreter //;t;d' /proc/sys/fs/binfmt_misc/qemu-arm
`
is the command i was looking for!